charts/thoras/templates/monitor-v2/deployment.yaml

```
@@ -0,0 +1,31 @@
+{{- if .Values.thorasMonitorV2.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: thoras-monitor-v2
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: thoras-monitor-v2
+  template:
+    metadata:
+      labels:
+        app: thoras-monitor-v2
+        app.kubernetes.io/name: {{ .Chart.Name }}
+        helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      serviceAccountName: thoras-monitor-v2
+      containers:
+        - name: thoras-monitor-v2
+          image: {{ .Values.imageCredentials.registry }}/thoras-monitor-v2:{{ default .Values.thorasVersion .Values.thorasMonitorV2.imageTag }}
```

[pebble] All of our current go services use the same `services` image. Unless there's a good reason to build them separately, I'd recommend using the same pattern here:
```suggestion
          image: {{ .Values.imageCredentials.registry }}/services:{{ default .Values.thorasVersion .Values.thorasMonitorV2.imageTag }}
```

Assuming you go this route, you'll also need to specify a `command: [...]` entry point. Otherwise the default `/app/api-server` will be started, which is probably not your intent.